libs/blocks/merch/merch.js

```diff
@@ -326,6 +325,10 @@ export async function getPriceContext(el, params) {
 export async function buildCta(el, params) {
   const large = !!el.closest('.marquee');
   const strong = el.firstElementChild?.tagName === 'STRONG' || el.parentElement?.tagName === 'STRONG';
+  const isUpgradeOffer = el.closest('.merch-offers.upgrade');
```

Comment:
Nit: you could save a line of code and move the statement into the `if` clause` since `isUpgradeOffer` is only used here.

Reply:
true, will fix